tests/nemo_text_processing/en/data_text_normalization/test_cases_money.txt

```diff
@@ -63,3 +63,4 @@ $1,925.21~one thousand nine hundred and twenty five dollars twenty one cents
 $1,234.123~one thousand two hundred and thirty four point one two three dollars
 US $76.3 trillion~US seventy six point three trillion dollars
 US$76.3 trillion~seventy six point three trillion us dollars
+The price for each canned salmon is $5, each bottle of peanut butter is $3~The price for each canned salmon is five dollars, each bottle of peanut butter is three dollars
```

Comment:
How did this one pass w/o Jenkins update?

Reply:
Interesting 🤔.

Reply:
seems like the added test case was working w/o weight update. Could you please replace with the input from [this issue](https://github.com/NVIDIA/NeMo-text-processing/issues/126)`Thank you for the quantities. Now, lets talk about the pricing. The price for each canned salmon is \$5, each bottle of peanut butter is \$3`